CMakePresets.json

```diff
@@ -7,12 +7,12 @@
       "binaryDir": "build-gcc",
       "cacheVariables": {
         "CMAKE_BUILD_TYPE": "Debug",
+        "USE_NATIVE_INSTRUCTIONS": "ON",
```

Comment:
Please stop changing unrelated things.
There's no reason to reorder these. All it does is add review overhead

Reply:
ok. I simply ordered all the 3 sections in the same way (they had almost all the same properties but in different order)